app/support/price_displayment.rb

```diff
@@ -82,6 +97,18 @@ def build_quantity_presenter
     QuantityPresenter.new(product, quantity_to_display)
   end
 
+  def calculated_billable_minutes
+    case reservation.price_policy.charge_for
+    when InstrumentPricePolicy::CHARGE_FOR.fetch(:reservation)
+      TimeRange.new(reservation.reserve_start_at, reservation.reserve_end_at).duration_mins
+    when InstrumentPricePolicy::CHARGE_FOR.fetch(:usage)
+      TimeRange.new(reservation.actual_start_at, reservation.actual_end_at).duration_mins
+    when InstrumentPricePolicy::CHARGE_FOR.fetch(:overage)
+      end_time = [reservation.reserve_end_at, reservation.actual_end_at].max
+      TimeRange.new(reservation.reserve_start_at, reservation.end_time).duration_mins
+    end
+  end
```

Comment:
Could you put this logic in `Reservation` instead and call it `billable_duration_mins`?

Then you'd have:
```
def calculated_billable_minutes
  set_billable_minutes? ? billable_duration_mins : nil
end
```

This allows us to reuse the logic and avoid having two methods named `calculated_billable_minutes` that do subtly different things